fix: style of recipe actions to be compliant with design schema

felixschndr opened the pull request:
## What this PR does / why we need it:

The design of the recipe actions in the three dot menu did not look good (not like the other points) and it was unclear that is is possible to click on the text to show the actions. This fixes it:
- The menu for actions is always visible now (if there is at least one recipe action 
- The text is aligned properly
- Removed the header
- The actions have an icon (always the same, the same one as in `/group/data/recipe-actions`), changed that icon as discussed below to `mdiLinkVariantPlus` 

------


Old design:
<img width="314" height="365" alt="image" src="https://github.com/user-attachments/assets/862eb953-abf4-47f1-a63b-bf5ebd20ed2e" />
<img width="315" height="443" alt="image" src="https://github.com/user-attachments/assets/28d5eaab-821f-47c2-99af-f686b6ba8f18" />



New design:
<img width="267" height="465" alt="image" src="https://github.com/user-attachments/assets/0ca4f844-9173-4061-88f4-bc1cc0e40a0d" />

<img width="343" height="205" alt="image" src="https://github.com/user-attachments/assets/19db06c2-468c-42d8-a67a-69ff0f4f1645" />


the menu is always visible now (if there is at least one recipe action and the text is aligned properly

## Which issue(s) this PR fixes:

Fixes https://github.com/mealie-recipes/mealie/issues/5734



## Testing

`task ui:test` runs successfully

felixschndr committed:
style: fix style of recipe actions to be compliant with design schema

- always show the actions
- align text properly
- add icons to actions

Kuchenpirat reviewed (commented):
Hey, thanks for the catch and the PR. 

I suggest using another icon. I dont think we should use the mealie icon for this. 

I would suggest: [mdiLinkVariantPlus](https://pictogrammers.com/library/mdi/icon/link-variant-plus/)

You can add new icons in the frontend/lib/icons/icons.ts file. 
Would also make sense to update the data management page then.

felixschndr commented:
Okay, I will change that.
I also suggest removing the header "Recipe actions" (just show the divider and then the actions) since the text is kinda out of place and not that _nice to look at_. What do you think about that?

Kuchenpirat commented:
yeah, makes sense to me.

Kuchenpirat commented:
@felixschndr if you have a better icon in mind or come across a better fitting one, im absolutely open for suggestions

felixschndr committed:
style: replaced the generic icon with mdiLinkVariantPlus for recipe actions

felixschndr commented:
> if you have a better icon in mind or come across a better fitting one, im absolutely open for suggestions

No, I really like the `mdiLinkVariantPlus`, I think it fits in great. 

I changed the icon and updated the design at the top of the MR. Thus, this MR is ready to be merged from my side :)

Kuchenpirat reviewed (changes requested):
Looks good so far, 
One minor thing that i catched. 

The recipe actions create and edit dialogs still use the primary icons.

felixschndr commented:
Good catch, fixed that

Kuchenpirat reviewed (changes requested):
please also remove the package-lock.json, package.json and yarn.lock from the pr.

felixschndr commented:
They slipped in there by accident, removed them
